variables.tf

```diff
@@ -83,9 +87,3 @@ variable "labels" {
   type        = map(string)
   default     = null
 }
-
-variable "enable_apis" {
-  description = "Enable required APIs for Cloud Memorystore."
-  type        = bool
```

Comment:
Don't remove the type or default.

Reply:
Okay, I included the type & default and matched it with the project services module: https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/project_services/variables.tf#L21-L24